client/src/components/dashboard/Cards.js

```diff
@@ -71,14 +66,18 @@ class Cards extends React.Component {
 Cards.propTypes = {
   document: PropTypes.object.isRequired,
   id: PropTypes.number.isRequired,
-  dispatch: PropTypes.func.isRequired
+  actions: PropTypes.object.isRequired
 };
 
-const mapStateToProps = (state, ownProps) => {
+const mapStateToProps = (state, ownProps) => ({
+  currentDoc: state.documents.currentDoc
+});
+
+function mapDispatchToProps(dispatch) {
   return {
-    currentDoc: state.documents.currentDoc
+    actions: bindActionCreators(docActions, dispatch)
```

Comment:
**Code quality**
You only have one use case for `bindActionCreators` which is when you want to pass `dispatch` down to a component that's not connected to the store.

`mapDispatchToProps` can also be an object
```javascript
const mapDispatchToProps = {
  getDoc,
  deleteDocument
}
```